sdk/schemaregistry/schema-registry/src/schemaRegistryClient.ts

```diff
@@ -56,14 +60,20 @@ export class SchemaRegistryClient implements SchemaRegistry {
       },
     };
 
-    this.client = new GeneratedSchemaRegistryClient(this.fullyQualifiedNamespace, {
+    this._client = new GeneratedSchemaRegistryClient(this.fullyQualifiedNamespace, {
       endpoint: this.fullyQualifiedNamespace,
       apiVersion: options.apiVersion,
       ...internalPipelineOptions,
     });
 
+    this._tracing = createTracingClient({
+      namespace: "Microsoft.EventHub",
```

Comment:
Is this the correct Resource Provider? I honestly wasn't sure - I looked here https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-services-resource-providers but didn't find schema-registry

Reply:
yes, it is currently hosted by event hubs